### PR TITLE
Add custom properties in the JMSSource config for custom JMS properties.

### PR DESCRIPTION
**Pull request by henry-cask**
Add custom properties in the JMSSource config for custom JMS properties since the UI does not have ability to add custom properties.

### CODE REVIEW

**Comment by henry-cask** [comment]
CC @albertshau 

Could you try this PR?

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java [hunk 1]
```diff
@@ -342,13 +355,18 @@ public JmsProvider getJmsProvider() {
     @Nullable
     public String jmsPluginType;
 
+    @Name(JMS_CUSTOM_PROPERTIES)
+    @Description("Provide any custom properties (such as Aggregations) as a JSON Map")
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java]: what are aggregations in jms?

**Comment by albertshau** [comment]
small question about the description, otherwise lgtm

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java (hunk 1 above)
Comment (henry-cask) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java]: Bad copy paste =P

Will update the description and merge it

**Comment by henry-cask** [comment]
Merging ...

**Pull request merged**